config/default.go

```diff
@@ -128,6 +128,7 @@ L1SyncCheckL2BlockNumberModulus = 600
 			AceptableInacctivityTime = "5s"
 			ApplyAfterNumRollupReceived = 10
 	[Synchronizer.L2Synchronization]
+		Enable = true
```

Comment:
It should be "Enabled" to be consistent with other "Enabled" config parameters that we have in other components (Pool, Sequencer, etc)

Reply:
changed on original PR and this one